A cyclist rides 3 km west and then turns around and rides 2 km east.

(a) What is their displacement? 

Δ𝑥 = 𝑥f − 𝑥0 = -1.0 km − 0 km = -1.0 km.

(b) What distance do they ride?

3 km + 2 km = 5 km

(c) What is the magnitude of their displacement?

1 km

A commuter train travels from Baltimore to Washington, DC, and back in 1 hour and 45 minutes. The distance between the two stations is approximately 40 miles. What is 

(a) the average velocity of the train

0

(b) the average speed of the train in m/s?

20.44 m/s

Average Speed

In [14]:
miles_per_minute = 80 / 105
miles_per_second = miles_per_minute / 60
meters_per_second = miles_per_second * 1609.34

print(round(meters_per_second, 2), 'm/s')

20.44 m/s


A racehorse coming out of the gate accelerates from rest to a velocity of 15.0 m/s due west in 1.80 s. What is its average acceleration?

In [2]:
avg_acc = -15 / 1.8

print(round(avg_acc, 2), "m/s^2")

-8.33 m/s^2


Suppose a train accelerates from rest to 30.0 km/h in the first 20.0 s of its motion. What is its average acceleration during that time interval?

In [22]:
tr_acc = 30 * 1000 / 3600 / 20

print(round(tr_acc, 2), "m/s^2")

0.42 m/s^2


Now suppose that at the end of its trip, the train slows to a stop from a speed of 30.0 km/h in 8.00 s. What is its average acceleration while stopping?

In [23]:
tr_acc = -30 * 1000 / 3600 / 8

print(round(tr_acc, 2), "m/s^2")

-1.04 m/s^2


A jogger runs down a straight stretch of road with an average velocity of 4.00 m/s for 2.00 min. What is his final position, taking his initial position to be zero?

In [6]:
# x = x0 + v * t
x = 0 + (4 * (2 * 60))

print(x, 'm')

480 m


An airplane lands with an initial velocity of 70.0 m/s and then decelerates at  1.50 m/s^2 for 40.0 s. What is its final velocity?

In [5]:
# v = v0 + a * t
v = 70 + (-1.5 * 40)

print(v, 'm/s')

10.0 m/s


Dragsters can achieve average accelerations of  26.0 m/s^2. Suppose such a dragster accelerates from rest at this rate for 5.56 s. How far does it travel in this time?

In [9]:
# x = x0 + v0*t + 1/2*a*t^2
x = 0 + (0 * 5.56) + (0.5 * 26 * (5.56**2))
print(round(x), ' m')

402  m


Calculate the final velocity of the dragster.

In [14]:
# v^2 = v0^2 + 2a(x - x0)

from math import sqrt
final_v = sqrt((2 * 26) * 402)
print(round(final_v), 'm/s')

145 m/s


On dry concrete, a car can decelerate at a rate of  7.00 m/s^2, whereas on wet concrete it can decelerate at only  5.00 m/s^2. Find the distances necessary to stop a car moving at 30.0 m/s (about 110 km/h) 

(a) on dry concrete and 

(b) on wet concrete. 

(c) Repeat both calculations, finding the displacement from the point where the driver sees a traffic light turn red, taking into account his reaction time of 0.500 s to get his foot on the brake.

In [37]:
w_a = -5
d_a = -7
r_t = 0.5
v = 30

w_t = -v/w_a
w_distance = ((v * w_t) + (0.5 * w_a * (w_t**2)))

d_t = -v/d_a
d_distance = ((v * d_t) + (0.5 * d_a * (d_t**2)))

r_w_distance = w_distance + (r_t * v)
r_d_distance = d_distance + (r_t * v)

print('wet distance:', round(w_distance))
print('dry distance:', round(d_distance))
print('reaction wet distance:', round(r_w_distance))
print('reaction dry distance:', round(r_d_distance))


wet distance: 90
dry distance: 64
reaction wet distance: 105
reaction dry distance: 79


Suppose a car merges into freeway traffic on a 200-m-long ramp. If its initial velocity is 10.0 m/s and it accelerates at  2.00 m/s^2, how long does it take to travel the 200 m up the ramp? (Such information might be useful to a traffic engineer.)

x = x0 + v0*t + 0.5*a*t^2

200 = 0 + 10 * t + 0.5 * 2 * t^2

200 = 10t + t^2

t^2 + 10t - 200 = 0

(t - 10) * (t + 20) = 0

t = 10 or t = -20

t = 10 s


A rocket accelerates at a rate of  20 m/s^2 during launch. How long does it take the rocket to reach a velocity of 400 m/s?

a = 20 m/s^2

v = 400 m/s

v0 = 0 m/s

<hr>

400 = 0 + 20t

20t = 400

t = 20

A person standing on the edge of a high cliff throws a rock straight up with an initial velocity of 13.0 m/s. The rock misses the edge of the cliff as it falls back to earth. Calculate the position and velocity of the rock 1.00 s, 2.00 s, and 3.00 s after it is thrown, neglecting the effects of air resistance.


In [55]:
seconds = [1, 2, 3]
# seconds = [round(x * 0.01, 2) for x in range(300)]
v0 = 13
g = 9.8

def v_and_y(v0, t, g):
    v = v0 - (g*t)
    y = 0 + (v0*t) - (0.5*g*(t**2))
    return round(v, 2), round(y, 2)

for s in seconds:
    v, y = v_and_y(v0, s, g)
    print(s*1.0, 's |', 'v', v, '| y', y)

1.0 s | v 3.2 | y 8.1
2.0 s | v -6.6 | y 6.4
3.0 s | v -16.4 | y -5.1


What happens if the person on the cliff throws the rock straight down, instead of straight up? To explore this question, calculate the velocity of the rock when it is 5.10 m below the starting point, and has been thrown downward with an initial speed of 13.0 m/s.

In [78]:
from math import sqrt
v0 = -13
y = -5.1
g = -9.8

v = -sqrt(v0**2 + ((2 * g) * y))
print(v)

-16.4


A chunk of ice breaks off a glacier and falls 30.0 meters before it hits the water. Assuming it falls freely (there is no air resistance), how long does it take to hit the water?

In [83]:
y = -30
g = -9.8
v0 = 0

v = -sqrt((v0**2) + ((2*g) * y))
t = (v0 + (v / g))

print(round(t, 2))


2.47
